caniusepython3/pypi.py

```diff
@@ -46,7 +46,7 @@ def just_name(supposed_name):
 
 @contextlib.contextmanager
 def pypi_client():
-    client = xmlrpc_client.ServerProxy('https://pypi.io/pypi')
+    client = xmlrpc_client.ServerProxy('https://pypi.python.org/pypi')
```

Comment:
Can you change this to pypi.org instead? I purposefully moved away from pypi.python.org for reliability concerns.